src/Tribe/Ignored_Events.php

```diff
@@ -50,12 +51,113 @@ private function __construct() {
 
 			add_action( 'wp_ajax_tribe_convert_legacy_ignored_events', array( $this, 'ajax_convert_legacy_ignored_events' ) );
 
+			// Modify Success messages
+			add_filter( 'bulk_post_updated_messages', array( $this, 'filter_updated_messages' ), 10, 2 );
+
 			/**
 			 * Register Notices
 			 */
 			tribe_notice( 'legacy-ignored-events', array( $this, 'render_notice_legacy' ), 'dismiss=1&type=warning' );
 		}
 
+		public function action_restore_ignored() {
```

Comment:
This whole File needs more Docblocks, I will do it later on this week. Sounds good?